sites/partners/src/lib/listings/BooleansFormatter.ts

```diff
@@ -41,12 +41,26 @@ export default class BooleansFormatter extends Formatter {
         this.data.whereApplicationsMailedIn === addressTypes.anotherAddress,
       trueCase: () => this.data.listingsApplicationMailingAddress,
     })
-    this.processBoolean("images", {
+    this.processBoolean("listingImages", {
       when:
         this.data.listingImages?.length &&
         !!this.data.listingImages[0].assets.fileId &&
         !!this.data.listingImages[0].assets.label,
-      trueCase: () => this.data.listingImages,
+      trueCase: () => {
+        return this.data.listingImages.map((listingImages) => {
+          return { ordinal: listingImages.ordinal, assets: listingImages.assets }
+        })
+      },
+    })
+    this.processBoolean("assets", {
+      when: !this.data.assets || this.data.assets.toString() === "{}",
+      trueCase: () => {
+        return []
+      },
+      falseCase: () => {
+        // TODO: figure out what is going on here
```

Comment:
Is there a ticket / more information we could add to this comment?

Reply:
ope, this was a comment for myself while working. I still need to figure this part out and will update it here when I do in fact figure it out